packages/react-jss/src/injectSheet.test.js

```diff
@@ -75,23 +75,6 @@ describe('React-JSS: injectSheet', () => {
       expect(document.querySelectorAll('style').length).to.be(0)
     })
 
-    it('should reuse one sheet for many elements wrapped into a JssProvider', () => {
-      render(
-        <div>
-          <JssProvider>
-            <MyComponent />
-          </JssProvider>
-          <JssProvider>
-            <MyComponent />
-          </JssProvider>
-        </div>,
-        node
-      )
-      expect(document.querySelectorAll('style').length).to.be(1)
-      unmountComponentAtNode(node)
-      expect(document.querySelectorAll('style').length).to.be(0)
-    })
```

Comment:
I had to remove this test. In my opinion, it doesn't make sense, because if you add a `classNamePrefix` to the first provider, the second provider with the styled components will also end up using the `classNamePrefix`

Example: https://codesandbox.io/s/r5j1ynwnwm

Reply:
Yeah, reusing styles from different providers will only work if they have identic settings and in that case why there are different providers.